drivers/flash/Kconfig.simulator

```diff
@@ -16,6 +16,15 @@ menuconfig FLASH_SIMULATOR
 
 if FLASH_SIMULATOR
 
+config FLASH_SIMULATOR_UNALIGNED_READ
+	bool
+	prompt "Allow read access to be unaligned"
+	default y
+	help
+	 If selected, reading operation would not check if the access is aligned.
```

Comment:
If selected, the reading operation does not check if access is aligned.
         Enable this option only if you want to simulate a specific FLASH interface
	 that requires alignment read access.

Reply:
I just saw that there is a logical mistake, should be:
**Disable** this option only...

Thank you. I will update it after we get shippable green.

Reply:
That suddenly makes a lot more sense. ;)

Reply:
Done.